docs/en/user_guides/3_pretrain.md

```diff
@@ -8,11 +8,11 @@
     - [Train with multiple machines](#train-with-multiple-machines)
     - [Launch multiple jobs on a single machine](#launch-multiple-jobs-on-a-single-machine)
 
-This page provides basic usage about how to run algorithms and how to use some tools in MMSelfSup. For installation instructions and date preparation, please refer to [install.md](install.md) and [prepare_data.md](prepare_data.md).
+This page provides the basic usage about how to run algorithms and how to use some tools in MMSelfSup. For installation instructions and data preparation, please refer to [get_started.md](../get_started.md) and [dataset_prepare.md](2_dataset_prepare.md).
 
 ## Start to Train
 
-**Note**: The default learning rate in config files is for specific number of GPUs, which is indicated in the config names. If using different number GPUs, the total batch size will change in proportion, you have to scale the learning rate following `new_lr = old_lr * new_ngpus / old_ngpus`.
+**Note**: The default learning rate in config files is for specific number of GPUs, which is indicated in the config names. If you use different number of GPUs, the total batch size will be changed in proportion. In this case, you have to scale the learning rate following `new_lr = old_lr * new_ngpus / old_ngpus`.
```

Comment:
better to explain the learning rate scaling by batch size, it is kind of confusing to explain it by 'number of gpus'